qlib/workflow/recorder.py

```diff
@@ -229,3 +234,11 @@ def get_artifact_uri(self, artifact_path=None):
         if self.artifact_uri is not None:
             return self.artifact_uri
         return mlflow.get_artifact_uri(artifact_path)
+

```

Comment:
Replace `delete_tag(self, key)` with `def delete_tags(self, *keys):`  will be more convinient